packages/ecs-agent/ecs-agent.spec

```diff
@@ -339,44 +192,39 @@ ln -rs %{buildroot}%{_cross_localstatedir}/ecs/managed-agents/execute-command/co
 install -d %{buildroot}%{ecs_exec_dir}/certs
 ln -rs %{buildroot}%{_cross_sysconfdir}/pki/tls/certs/ca-bundle.crt %{buildroot}%{ecs_exec_dir}/certs/tls-ca-bundle.pem
 
-# Prepare license and vendor information so it can be co-installable
-mv %{ecscni_gorepo}-%{ecscni_gitrev}/LICENSE %{ecscni_gorepo}-%{ecscni_gitrev}/LICENSE.%{ecscni_gorepo}
-mv %{vpccni_gorepo}-%{vpccni_gitrev}/LICENSE %{vpccni_gorepo}-%{vpccni_gitrev}/LICENSE.%{vpccni_gorepo}
-# Move vendor folder into a single directory so cross_scan_attribution can run once
-mkdir go-vendor
-mv %{agent_gorepo}-%{agent_gover}/agent/vendor go-vendor/%{agent_gorepo}
-mv %{ecscni_gorepo}-%{ecscni_gitrev}/vendor go-vendor/%{ecscni_gorepo}
-mv %{vpccni_gorepo}-%{vpccni_gitrev}/vendor go-vendor/%{vpccni_gorepo}
-%cross_scan_attribution go-vendor go-vendor
+install -d %{buildroot}{%{_cross_libexecdir},%{_cross_fips_libexecdir}}/amazon-ecs-agent
+# Create symlinks to ECS CNI plugin binaries
+for p in \
+  ecs-bridge \
+  ecs-eni \
+  ecs-ipam \
+; do
+  ln -rs %{buildroot}%{_cross_libexecdir}/cni/ecs/${p} %{buildroot}%{_cross_libexecdir}/amazon-ecs-agent/${p}
+  ln -rs %{buildroot}%{_cross_libexecdir}/cni/ecs/${p} %{buildroot}%{_cross_fips_libexecdir}/amazon-ecs-agent/${p}
+done
+
+# Create symlinks to VPC CNI plugin binaries
+for p in \
+  aws-appmesh \
+  ecs-serviceconnect \
+  vpc-branch-eni \
+  vpc-bridge \
+  vpc-eni \
+  vpc-tunnel \
+; do
+  ln -rs %{buildroot}%{_cross_libexecdir}/cni/vpc/${p} %{buildroot}%{_cross_libexecdir}/amazon-ecs-agent/${p}
+  ln -rs %{buildroot}%{_cross_libexecdir}/cni/vpc/${p} %{buildroot}%{_cross_fips_libexecdir}/amazon-ecs-agent/${p}
+done
```

Comment:
It's very strange for a package to symlink another package's binaries like this, and to vend those symlinks. I acknowledge that it was done for the SSM agent plugin but it's not a great pattern to emulate. It would be better to create these links in the corresponding CNI package. You could put them in an "ecs" subpackage, but that's not as important as keeping the links and binaries together.

Some concerns with this approach:
* If a new VPC or ECS plugin is added, this package definition must also change in order to create symlinks. Where if the plugin package also owns the symlink, then it will be more obvious to the packager that this dependency exists.
* This will create broken symlinks, if the plugin package drops a plugin or changes the path where plugins are installed.

Reply:
Ack regarding dropping `pkg`* from commit messages - will fix those. Regarding rationale, I put that in the issue #90 : 

```
Ideally, we want to split these plugins into their own packages for ease of building and patching.
```

But I'll copy that over to this PR description!

Reply:
Good callouts regarding the symlinks. My (naive) reasoning was that since they are being installed in the `/amazon-ecs-agent` directory, they should be added to the `amazon-ecs-agent` spec, but I agree this will cause more maintenance issues in the future. Will create these symlinks in their respective pkg as suggested.